packages/openapi3/src/openapi.ts

```diff
@@ -285,6 +285,7 @@ function createOAPIEmitter(
     allHttpAuthentications: HttpAuth[],
     defaultAuth: AuthenticationReference,
     version?: string,
+    moduleContent?: any,
```

Comment:
see other comment in `xml-module`, I think its better to catch the load of the xml library not the module in case there error there.